docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md

```diff
@@ -68,11 +68,11 @@ The following video walks you through the process of sideloading your add-in in
 
 ### Configure the trust with a Registry script
 
-1. In a text editor, create a file named TrustNetworkShareCatalog.reg. 
+1. In a text editor, create a file named TrustNetworkShareCatalog.reg.
 
 2. Add the following content to the file:
 
-	```
+    ```
```

Comment:
The indentation looks off here.

Reply:
Good catch. Tabs vs spaces, with the linter still complaining about the unspecified codeblock language :-/